package.json

```diff
@@ -2,6 +2,8 @@
   "name": "graphprotocol-tools-monorepo",
   "private": true,
   "devDependencies": {
+    "@changesets/changelog-github": "^0.4.7",
+    "@changesets/cli": "^2.25.2",
```

Comment:
Maybe being picky here, but what do you guys think of locking versions? We had an issue in the past with security related to a dependency and then we decided to start to pin them.

Reply:
these are dev dependencies and we keep `renovate` bot to keep them up to date